packages/teleterm/src/sharedProcess/ptyHost/ptyProcess.ts

```diff
@@ -178,5 +178,7 @@ async function getWorkingDirectory(pid: number): Promise<string> {
     case 'linux':
       const asyncReadlink = promisify(readlink);
       return await asyncReadlink(`/proc/${pid}/cwd`);
+    case 'win32':
+      return undefined;
```

Comment:
Did you try to implement it for Windows? What problems did you run into?

Reply:
This whole function looks rather complicated. Does `process.cwd()` not work?

Reply:
>This whole function looks rather complicated. Does process.cwd() not work?

No, `process.cwd()` returns only the directory where the process was spawned. 
The method from above is used also by `vscode` or `tabby`.

>Did you try to implement it for Windows? What problems did you run into?

Hmm is there any way to do it? The apps I mentioned don't support CWD for Windows. From what I see, there is a [Get-Process](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-process?view=powershell-7.2) command, but it doesn't return a working directory.

Reply:
> Hmm is there any way to do it?

I don't know, that's why I asked. 😏 If there's no easy way to do that then it's fine I guess. I haven't installed Parallels yet so I cannot check if cmd.exe or Powershell show something (if anything) instead of cwd.